app/api/routes.py

```diff
@@ -106,9 +104,9 @@ def get_resource(id):
 
     finally:
         if resource:
-            return standardize_response(resource.serialize, None, "ok")
+            return standardize_response(payload=dict(data=(resource.serialize)))
         else:
-            return standardize_response({}, None, "ok")
+            return standardize_response(payload=dict(data={}))
```

Comment:
I realized a while ago that this is actually incorrect, we should return a 404 if the resource is not found in the DB. Do you think you can go ahead and update that while you're in there?